source/docs/3/basic_filtering_and_styling_with_expressions.rst

```diff
@@ -0,0 +1,120 @@
+Basic Filtering and Styling with Expressions (QGIS3)
```

Comment:
The url is too long. Make the slug `vector-styling-expressions.rst` by renaming this file.